subsys/usb/usb_c/usbc_pe_common.c

```diff
@@ -898,6 +898,9 @@ static void pe_send_not_supported_entry(void *obj)
 
 	LOG_INF("PE_Not_Supported");
 
+	/* Notify the Device Policy Manager of unsupported message reception */
+	policy_notify(dev, MSG_NOT_SUPPORTED_RECEIVED);
```

Comment:
This is hard notification to name since we're sending a Not_supported message to say we received a not supported message.... how about UNSUPPORTED_MSG_RECEIVED?